src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php

```diff
@@ -17,6 +17,9 @@
 use Symfony\Component\Config\Resource\ClassExistenceResource;
 use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
 
+/**
+ * @internal since Symfony 7.1
```

Comment:
I'm not sure about this one, the class can be useful for third party bundles. We already made a method final/internal so we already closed the class a bit.

Reply:
Should we mark the `warmUp()` method as final instead?

Reply:
I'm not sure it would provide much benefits doing so.

Reply:
up to revert this line @xabbuh?

Reply:
done